runtime/moonriver/src/lib.rs

```diff
@@ -298,6 +304,7 @@ impl frame_system::Config for Runtime {
 	type PreInherents = ();
 	type PostInherents = ();
 	type PostTransactions = ();
+	type ExtensionsWeightInfo = ();
```

Comment:
Seems important, we should benchmark it.

Reply:
👍  something like [this](https://github.com/moondance-labs/tanssi/pull/906/files)?

Reply:
Yes, we need to add `[frame_system_extensions, frame_system_benchmarking::extensions::Pallet::<Runtime>]` to the define_benchmarks and then use the generated file in the config.

Reply:
Address https://github.com/moonbeam-foundation/moonbeam/pull/3224/commits/15fc6b380c8e7ab7a2127e167a1ffadf3ba55023